pkg/cloudprovider/providers/azure/azure_controllerCommon.go

```diff
@@ -63,8 +61,6 @@ type controllerCommon struct {
 	managementEndpoint    string
 	tokenEndPoint         string
 	aadResourceEndPoint   string
```

Comment:
clientID, clientSecret, managementEndpoint, tokenEndPoint, aadResourceEndPoint are also not in use, consider removing them all.